src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
 class Herbivore extends Animal {
-  // write your code here
-}
+  constructor(name, health, hidden = false) {
```

Comment:
The `health` parameter in the `Herbivore` constructor is unnecessary since the `Animal` class already provides a default value of `100` for health. You can remove it to simplify the constructor.